src/pages/verify/e-ids/danish-mitid-erhverv.mdx

```diff
@@ -74,11 +74,11 @@ Detailed instructions, along with a [video demonstration](#video), are provided
 1. Go to the test organization creation page and fill in the form: [https://testportal.test-devtest4-nemlog-in.dk/](https://testportal.test-devtest4-nemlog-in.dk/)
 
 2. The wizard will generate an organization admin account with username/password credentials. The password is self-supplied, and the username is autogenerated.  
-You can then follow the [Link til MitID Erhverv - IntTest](https://test-devtest4-nemlog-in.pp.mitid.dk/login/mitid) to login as the organization admin: 
+You can then follow the **Link til MitID Erhverv - IntTest** to login as the organization admin: 
```

Comment:
Do we not have the new link? Or does the sentence just reference a UI element?

Reply:
The sentence just references a UI element. I think it makes sense since you have to go through all the steps when creating a test user. And there won't be broken links in case it changes again.
![Screenshot 2024-04-05 at 12 41 37](https://github.com/criipto/docs/assets/44656084/43ff2251-8f26-4879-9ba7-b3dd958eec8d)